test/helper/distance.go

```diff
@@ -0,0 +1,22 @@
+package testhelper
```

Comment:
if this helper is only used for tests I would recommend changing it's name from: `distance.go` to `distance_test.go`